subsys/partition_manager/CMakeLists.txt

```diff
@@ -47,6 +47,10 @@ if (CONFIG_BSD_LIBRARY)
   ncs_add_partition_manager_config(pm.yml.bsdlib)
 endif()
 
+if (CONFIG_BT_RPMSG_NRF53)
```

Comment:
I feel like I start to see a pattern here.
I believe it could be wise to look into refactoring all those `pm.yml` handling in future.

Reply:
I agree, I've created a task for it.